FreeRTOS/Test/CMock/smp/single_priority_no_timeslice/single_priority_no_timeslice_utest.c

```diff
@@ -617,7 +619,7 @@ void test_task_create_tasks_lower_priority( void )
  * 
  * Task N - 1        New Task
  * Priority – 1      Priority – 2
- * State - Ready	 State - Running (Core N)
+ * State - Ready	 State - Running (First available core)
```

Comment:
JAMA should be updated to match this change

Reply:
Update in JAMA.